src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,18 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
     public static void main(String[] args) {
+        Logger logger = LogManager.getLogger(Main.class);
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = new User("Bob", "1234");
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(user.getLogin(), user.getPassword());
```

Comment:
I think it's redundant to create user object here, as you will receive it from our fake DB (method findByLogin). As a result, you only need to pass some params